src/pages/tickets.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<!--head section-->
+<head>
+  <title>Shapeland - Tickets</title>
+  <meta charset="utf-8">
+  <meta name="description" content="Practice web page.">
+  <meta name="author" content="shapeland">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="refresh" content="800">
+  <meta name="keywords" content="uol theme park">
+  <link rel="stylesheet" type="text/css" href="../css/main.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"> <!--social favigon-->
+  <link rel="stylesheet" type="text/css" href="../css/tickets.css">
+</head>
+
+<!--Ticket page body-->
+<body>
+    <!--header section-->
+	
+    <!--navbar--> 
+    <header class="sticky" style="z-index: 9999;"> 
+        <nav id="navbar">
+            <nav class="nav-container">
+                <div class="logo-container"><img src="../public/images/shape_logo_circ.png" class="logo"></div>
+                <div class="menu-container"><b>Attractions &amp; Facilities</b></div>
+                <div class="menu-container"><b>Ticket info</b></div>
+            </nav>
+        </nav>
+    </header>
+	
+	<!--hero img-->
+	<header class="hero-image">
+		<div class="hero-text">
```

Comment:
Maybe try to use  a `span`  here? We also might be better off using a `flex`  container on `hero-image`  and then remove the `display: absolute`  and try to position it only with flex properties.